tools/src/main/java/org/apache/rocketmq/tools/command/ha/GetSyncStateSetSubCommand.java

```diff
@@ -123,13 +123,23 @@ private void printData(String controllerAddress, List<String> brokerNames,
                     next.getKey(), next.getValue().getMasterAddress(), next.getValue().getMasterEpoch(), next.getValue().getSyncStateSetEpoch(),
                     inSyncReplicas.size());
                 for (BrokerReplicasInfo.ReplicaIdentity member : inSyncReplicas) {
-                    System.out.printf("\n InSyncReplica:\t%s\n", member.toString());
+                    System.out.printf("\n InSyncReplica:\t%s\n", member.toString() + acquireBrokerStatus(defaultMQAdminExt, member.getAddress()));
                 }
 
                 for (BrokerReplicasInfo.ReplicaIdentity member : notInSyncReplicas) {
-                    System.out.printf("\n NotInSyncReplica:\t%s\n", member.toString());
+                    System.out.printf("\n NotInSyncReplica:\t%s\n", member.toString() + acquireBrokerStatus(defaultMQAdminExt, member.getAddress()));
```

Comment:
Format the message in `printf` formatter instead of concatenating strings directly.

Reply:
i have fixed it